discoveries/coastal-flooding-and-slr.discoveries.mdx

```diff
@@ -53,13 +59,15 @@ thematics:
       zoom={1}
       center={[0,0]}
     />
-    <Caption 
-      attrAuthor='NASA' 
+    <Caption
+      attrAuthor='NASA'
       attrUrl='https://nasa.gov/'
     />
   </Figure>
   <Prose>
-  Identifying hot spot regions under non-stationary water cycle changes and exploring the drivers are important issues that may provide information to improve hydrological forecasting and water management. Through interdisciplinary research and data assimilation, the EIS team developed a **novel metric to quantify the non-stationarity of the global water cycle**, integrating data about long term trends, seasonal shifts, and interannual to sub-seasonal scale variabilities.
+  Identifying hot spot regions under non-stationary water cycle changes and exploring the drivers are important issues that may provide information to improve hydrological forecasting and water management. Through interdisciplinary research and data assimilation, the EIS team developed a **novel metric to quantify the non-stationarity of the global water cycle**, integrating information on long term trends, seasonal shifts, and changes in the occurrence of extreme events.
+
+  Learn more about the non-stationarity metric <u><a href="https://github.com/Earth-Information-System/sea-level-and-coastal-risk/blob/main/AMS_2023_Wanshu_Nie_for_VEDA_Discovery.pdf">here</a></u>.
```

Comment:
For accessibility purposes is generally good practice to avoid "here" links as they lack context.

A possible alternative:
```suggestion
  This linked document contains [more information about the non-stationarity metric](https://github.com/Earth-Information-System/sea-level-and-coastal-risk/blob/main/AMS_2023_Wanshu_Nie_for_VEDA_Discovery.pdf).
```